File: CPE-1star/UVA11461-Square Numbers .py
#判斷範圍內的完全平方數有幾個
import math
while True:
	try:
		a,b=map(int,input().split())
		if a==0 and b==0:
			break
		count=0
		for i in range(a,b+1):
			temp=math.sqrt(i)
			if temp-int(temp)==0:
				count+=1
		print(count)
	except:
		break

File: CPE-1star/UVA11461-Square Numbers.cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;
 
int main()
{
    int a,b;
    while(cin>>a>>b){
        if(a==0 & b==0)
            break;
        int count=0;
        for(int i=a;i<=b;i++){
            float temp=sqrt(i);
            if((temp-(int)temp)==0)
                count+=1;
        }
        cout<<count<<'\n';
    }
    return 0;
}